src/components/screens/monitor/delegates/delegates.test.js

```diff
@@ -6,6 +6,7 @@ import delegates from '../../../../../test/constants/delegates';
 const transformToLiskServiceFormat = ({ account, ...delegate }) => ({
   ...delegate,
   ...account,
+  status: 'forgedThisRound',
```

Comment:
If they all have the same status, why bother set it there?

Reply:
we can keep this based on what we discuss